packages/swapper/src/swappers/utils/filterBuyAssetsBySellAssetId/filterBuyAssetsBySellAssetId.ts

```diff
@@ -36,3 +49,11 @@ export const filterCrossChainEvmBuyAssetsBySellAssetId = (
     buyAssetChainId !== sellAssetChainId
   return _filterEvmBuyAssetsBySellAssetId(input, crossChainIdPredicate)
 }
+
+export const filterSameChainSolanaBuyAssetsBySellAssetId = (
+  input: BuyAssetBySellIdInput,
+): Asset[] => {
+  const sameChainIdPredicate = (buyAssetChainId: ChainId, sellAssetChainId: ChainId): boolean =>
+    buyAssetChainId === sellAssetChainId
+  return _filterSolanaBuyAssetsBySellAssetId(input, sameChainIdPredicate)
+}
```

Comment:
And this one should be removed too after tackling `filterBuyAssetsBySellAssetId` suggested changes

Reply:
Removed in [4e76e81](https://github.com/shapeshift/web/pull/8120/commits/4e76e8115223d91bd602f7d9ceed80c94c77ea15)